app/modules/eus/http_html_backup.def

```diff
@@ -1,205 +1,213 @@
 static const char http_html_backup[] = {
```

Comment:
Again, changes should not be part of the default. Same goes for `http_html_backup.def`.